lighthouse-core/gather/gather-runner.js

```diff
@@ -626,12 +628,15 @@ class GatherRunner {
    * will have the reason. See manifest-parser.js for more information.
    *
    * @param {LH.Gatherer.PassContext} passContext
-   * @return {Promise<LH.Artifacts.Manifest|null>}
+   * @return {Promise<{parsedManifest: LH.Artifacts.Manifest|null, manifestUrl: string|null} | null>}
    */
   static async getWebAppManifest(passContext) {
     const response = await passContext.driver.getAppManifest();
     if (!response) return null;
-    return manifestParser(response.data, response.url, passContext.url);
+    const parsedManifest = manifestParser(response.data, response.url, passContext.url);
+    const manifestUrl = response.url;
```

Comment:
we talked in voice about this.. should manifestUrl be part of the parser's return value?  I said no, but... let's reconsider. ... it could. and it kinda makes sense. just spoke with @brendankenny and we both feel fairly ambivalent about it.

It keeps things a tad cleaner to put it in the manifestParser return, so let's do that. (well, as it turns out... you [already did](https://github.com/GoogleChrome/lighthouse/pull/11330/files#diff-af5212b94825d538f81ee91d73c6e326))

that actually means you can revert allllll the changes in gather-runner

anddddddddddd i think everything will just still end up working fine.

Reply:
yea it seems like this whole file could be reverted and the url from the parser lib will be consumable in `installable-manifest`